fineract-provider/build.gradle

```diff
@@ -59,9 +61,83 @@ apply plugin: 'com.bmuschko.tomcat'
 apply plugin: 'project-report'
 apply plugin: 'java'
 apply plugin: 'openjpa'
+
 // apply plugin: 'pmd'
 // apply plugin: 'findbugs'
 
+dependencyManagement {
+  imports {
+    mavenBom 'org.springframework:spring-framework-bom:5.1.10.RELEASE'
+  }
+  dependencies {
```

Comment:
I'm probably misunderstanding the Spring BOM, but does it not also define (some of) these versions? Or is this list just the versions of all Fineract dependencies which the Spring BOM does not fix?

Reply:
Well, the spring framework-bom resolves all spring artifacts(like spring-core etc) without explicitly declaring them since they all use thesame version

It seems there is no spring-boot-bom to handle the spring boot artifacts and dependencies

Reply:
My primary idea was to use generalized bom that handles dependencies for most of the packages we use and even more but the version conflicts were too big breaking builds

Reply:
> but the version conflicts were too big breaking builds

@xurror OK, then let's do that (you or someone else later) separately; I've just created new https://issues.apache.org/jira/browse/FINERACT-805 so that we don't forget to follow-up on that... (It's surely best to raise small PRs for each one of these, instead of one big one for everything together.)